packages/mobile/src/tokens/reducer.ts

```diff
@@ -38,12 +38,14 @@ export interface State {
   tokenBalances: StoredTokenBalances
   loading: boolean
   error: boolean
+  lastSuccessfulFetch: number
```

Comment:
nit: For correctness and avoiding a migration:
```suggestion
  lastSuccessfulFetch?: number
```